index.js

```diff
@@ -0,0 +1,120 @@
+const Alexa = require('ask-sdk-core');
+
+const randomDessert = ['Boston Creme Pie', 'Ice Cream', 'Cake', 'Cheesecake']
+const randomDessertGenerator = Math.floor(Math.random() * randomDessert.length)
+const randomDessertAnswer = randomDessert[randomDessertGenerator]
+
+//const randomDessert = {['Boston Creme Pie', 'Ice Cream', 'Cake', 'Cheesecake']
+
+const CancelAndStopRequestHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
+            (handlerInput.requestEnvelope.request.intent.name === 'AMAZON.CancelIntent' ||
+                handlerInput.requestEnvelope.request.intent.name === 'AMAZON.StopIntent')
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak('Enjoy your dessert!')
+            .withShouldEndSession(true)
+            .getResponse()
+    },
+}
+
+const FallbackIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
+            handlerInput.requestEnvelope.request.intent.name === 'AMAZON.FallbackIntent'
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak("I'm sorry, I didn't understand that. Do you want a dessert?")
+            .reprompt("Would you like the dessert of the day?")
```

Comment:
Be careful of Yes/No questions, you'll need to have YesIntent and NoIntent to handle the user saying yes to this otherwise it will trigger this FallbackIntent and the user will be in a loop.